src/policy/feerate.h

```diff
@@ -24,12 +24,13 @@ enum class FeeEstimateMode {
 };
 
 /**
- * Fee rate in satoshis per kilobyte: CAmount / kB
+ * Fee rate in satoshis per kilovirtualbyte: CAmount / kvB
```

Comment:
fwiw this is the first use of the word "kilovirtualbyte", do we use another term normally ?

Reply:
sounds good to me then, i don't mind using it here, just think unit consistency is important so was worried for a bit

Reply:
Amended all mentions of `size` and `bytes` in this file to use `vsize` and `vbytes`.